runtime/help/plugins.md

```diff
@@ -359,67 +417,66 @@ end
-list of which functions are supported you can look through `lua.go`
+list of functions that are supported, you can look through `lua.go`
 (which should be easy to understand).
 
-```
-fmt
-io
-io/ioutil
-net
-math
-math/rand
-os
-runtime
-path
-filepath
-strings
-regexp
-errors
-time
-unicode/utf8
-archive/zip
-net/http
-```
-
-For documentation for each of these functions, see the Go standard
-library documentation at https://golang.org/pkg/ (for the packages
-exposed to micro plugins). The Lua standard library is also available
-to plugins though it is rather small.
-
-The following functions are also available from the go-humanize package:
-
-The `humanize` package exposes:
-* `Bytes`
-* `Ordinal`
+* [fmt](https://pkg.go.dev/fmt)
+* [io](https://pkg.go.dev/io)
+* [io/ioutil](https://pkg.go.dev/io/ioutil)
+* [net](https://pkg.go.dev/net)
+* [math](https://pkg.go.dev/math)
+* [math/rand](https://pkg.go.dev/math/rand)
+* [os](https://pkg.go.dev/os)
+* [runtime](https://pkg.go.dev/runtime)
+* [path](https://pkg.go.dev/path)
+* [filepath](https://pkg.go.dev/filepath)
+* [strings](https://pkg.go.dev/strings)
+* [regexp](https://pkg.go.dev/regexp)
+* [errors](https://pkg.go.dev/errors)
+* [time](https://pkg.go.dev/time)
+* [unicode/utf8](https://pkg.go.dev/unicode/utf8)
+* [archive/zip](https://pkg.go.dev/archive/zip)
+* [net/http](https://pkg.go.dev/net/http)
+
+The following functions from the go-humanize package are also available:
+
+* `humanize`:
+    - `Bytes(s uint64) string`: produces a human readable representation of
+       an SI size.
+    - `Ordinal(x int) string`: gives you the input number in a rank/ordinal
+       format.
+
+[The Lua standard library](https://www.lua.org/manual/5.1/manual.html#5) is also
+available to plugins, though it is rather small.
 
 ## Adding help files, syntax files, or colorschemes in your plugin
 
 You can use the `AddRuntimeFile(name string, type config.RTFiletype,
                                 path string)`
 function to add various kinds of files to your plugin. For example, if you'd
 like to add a help topic to your plugin called `test`, you would create a
-`test.md` file, and call the function:
+`test.md` file and call the function:
 
 ```lua
 config = import("micro/config")
 config.AddRuntimeFile("test", config.RTHelp, "test.md")
 ```
 
 Use `AddRuntimeFilesFromDirectory(name, type, dir, pattern)` to add a number of
-files to the runtime. To read the content of a runtime file use
+files to the runtime. To read the content of a runtime file, use
 `ReadRuntimeFile(fileType, name string)` or `ListRuntimeFiles(fileType string)`
 for all runtime files. In addition, there is `AddRuntimeFileFromMemory` which
 adds a runtime file based on a string that may have been constructed at
 runtime.
 
 ## Default plugins
 
-There are 6 default plugins that come pre-installed with micro. These are
+There are 7 default plugins that come pre-installed with micro. These are
```

Comment:
```suggestion
The following plugins come pre-installed with micro:
```

Otherwise the chance is high to miss that number again, when we modify the length of the list.

Sorry for my very late response here.
I came by to merge that PR due to @dmaluka's hint.

Reply:
But in general thank you for that great rework!

Reply:
No worries! Thanks for taking the time to check it out. 

>Otherwise the chance is high to miss that number again, when we modify the length of the list.
Right, makes sense. On top of that, the number itself is not a very valuable piece of info (especially since the list is right below it).